default.py

```diff
@@ -25,139 +23,309 @@
+    filtered_urls = filter(lambda x: utils.parse_resolution_of_source(x[0]) <= prefereResSetting, fetched_urls)
+    return sorted(filtered_urls, key=lambda x: utils.parse_resolution_of_source(x[0]), reverse=True)
+
 
-    autoplay = True if 'true' in control.getSetting('autoplay') else False
+MENU_ITEMS = [
+    ("Latest", "latest_d"),
+    ("Newest", "newest_d"),
+    ("Recent Subbed", "recent_subbed_d"),
+    ("Popular Subbed", "popular_subbed_d"),
+    ("Recent Dubbed", "recent_dubbed_d"),
+    ("Popular Dubbed", "popular_dubbed_d"),
+    ("Genres", "genres"),
+    ("Search", "search_d"),
+    ("Settings", "settings")
+]
 
+
+@plugin.route("/latest", name="latest_d", options={"page": "1"})
+@plugin.route("/latest/<page>")
+def latest(page):
+    plugin.log.info(page)
+    animes = _BROWSER.get_latest(int(page))
+    plugin.log.info(animes[0])
+    plugin.log.info(animes[-1])
+    items = []
+    for anime in animes:
+        items.append(
+            {
+                "label": anime["name"],
+                "path": create_url(anime["url"], anime["name"]),
+                "context_menu": [("Download", actions.background(
+                    create_url(anime["url"], anime["name"], True)))]
+            }
+        )
+    return plugin.finish(items)
+
+
+@plugin.route("/animes/<anime>")
+def animes(anime):
+    plugin.log.info(anime)
+    anime_name = plugin.request.args["anime_name"][0]
+    download = "True" == plugin.request.args["download"][0]
+    if download:
+        dm.download_anime(plugin, _BROWSER, anime, anime_name)
+    # plugin.log.info(anime_name)
+    # print(anime_name)
+    plugin.log.info(anime_name)
+    episodes = _BROWSER.get_anime_episodes(anime, isDirectoryStyle())
+    plugin.log.info(episodes[0])
+    plugin.log.info(episodes[-1])
+    items = []
+    for episode in episodes:
+        items.append(
+            {
+                "label": episode["name"],
+                "path": create_url(episode["url"], anime_name)
+            }
+        )
+    return plugin.finish(items)
+
+
+@plugin.route("/play/<anime>/<episode_number>")
+def play(anime, episode_number):
+    anime_name = plugin.request.args["anime_name"][0]
+    plugin.log.info("Name: " + anime_name)
+    plugin.log.info("Anime: {} Episode: {}".format(anime, episode_number))
+    sources = _BROWSER.get_episode_sources(anime, int(episode_number))
+    serverChoice = filter(lambda x: plugin.get_setting(x[0]) == 'true', SERVER_CHOICES.iteritems())
+    serverChoice = map(lambda x: x[1], serverChoice)
+    sources = filter(lambda x: x[0] in serverChoice, sources)
+    autoplay = True if 'true' in plugin.get_setting('autoplay') else False
     s = SourcesList(sources, autoplay, sortResultsByRes, {
-        'title': control.lang(30100),
-        'processing': control.lang(30101),
-        'choose': control.lang(30102),
-        'notfound': control.lang(30103),
+        'title': "Fetching Sources",
```

Comment:
please use strings from string table..
this plugin should support multi-langugue